api/lambdas/sqs-to-converter/index.js

```diff
@@ -350,11 +350,7 @@ export const handler = Sentry.AWSLambda.wrapHandler(async event => {
           });
           await sendToDLQ(message);
 
-          const cxId = message.messageAttributes?.cxId?.stringValue;
-          const patientId = message.messageAttributes?.patientId?.stringValue;
-          const jobId = message.messageAttributes?.jobId?.stringValue;
-
-          if (cxId && patientId && jobId && isSidechainConnector()) {
+          if (cxId && patientId && isSidechainConnector()) {
```

Comment:
`jobId` was not required, and can only send update to API if the ids get parsed, so might as well use the parsed ones